src/elements/content-preview/README.md

```diff
@@ -33,23 +33,24 @@
+| contentOpenWithProps | Object | `{}` | Props that can be forwarded to the Content Open With UI Element. *See them in the props section of [Content Open With UI Element](#content-open-with-documentation)*                |
+| contentSidebarProps  | Object | `{}` | Props that can be forwarded to the Content Sidebar UI Element. *See them in the props section of [Content Sidebar UI Element](#content-sidebar-documentation)*                      |
+| hasHeader            | boolean | `true` | Visually hides the preview header if this is set to `false`.                                                                                                                        |
+| language             | string | `en-US` | *See the [Internationalization](../README.md#internationalization) section*                                                                                                         |
+| messages             | Map<string, string> |  | *See the [Internationalization](../README.md#internationalization) section*                                                                                                         |
+| onClose              | function |  | Callback function for when the file preview closes. If absent, the close button will not render in the header.                                                                      |
+| onLoad               | function |  | Callback function for when a file preview loads.                                                                                                                                    |
+| requestInterceptor   | function | | *See the [developer docs](https://developer.box.com/docs/box-content-preview#section-options).*                                                                                     |
+| responseInterceptor  | function | | *See the [developer docs](https://developer.box.com/docs/box-content-preview#section-options).*                                                                                     |
+| sharedLink           | string |  | *See the [developer docs](https://developer.box.com/docs/box-content-preview#section-options).*                                                                                     |
+| sharedLinkPassword   | string |  | *See the [developer docs](https://developer.box.com/docs/box-content-preview#section-options).*                                                                                     |
+| showAnnotations      | boolean | `true` | *See the [developer docs](https://developer.box.com/docs/box-content-preview#section-options).*                                                                                     |
+| shouldProvide        | boolean | `true` | Decide if it should wrap the element with the `Notification` and `Tooltip` Provider. Set this to false if you already wrap your app with both of these providers.                       |
```

Comment:
if this is a boolean then maybe `hasProviders`?

---

alternate approach:
or what about an array/object that specifies providers? that way it's more configurable depending on the providers that the consumer already has

for example:

```
// in consuming app
const providers = ['date-picker', 'notification', 'tooltip']; // Blueprint DatePicker has a provider we'll eventually need to include

<ContentPreview providers={providers} ... />


// in new providers component
const ElementProviders = ({ children, providers = ['notification', 'tooltip'] }) => {
    let element = children;

    providers.forEach(provider => {
        switch (provider) {
            case 'date-picker':
                ...
                break;
            case 'notification':
                element = <Notification.Provider>...</Notification.Provider>;
                break;
            case 'tooltip':
                element = <TooltipProvider>{element}</TooltipProvider>;
                break;
        }
    });

    return element;
};
```